Battleship/Battleship/Game.cs

```diff
@@ -1,80 +0,0 @@
-﻿using System;
-
-namespace Battleship
-{
-    public class Game
-    {
-        public Player Player1 { get; set; }
-        public Player Player2 { get; set; }
-        private int roundCount { get; set; }
-
-        public Game()
-        {
-            Player1 = new Player("Player 1");
-            Player2 = new Player("Player 2");
-            Player1.SetShips();
-            Player2.SetShips();
-            Console.WriteLine("Player 1 sips:");
-            Player1.ShowBoards();
-            Console.WriteLine("Player 2 sips:");
-            Player2.ShowBoards();
-        }
-
-        public void PlayGame()
-        {
-            while (!Player1.DidLost && !Player2.DidLost)
```

Comment:
Could be introduced method GameIsFinished for while condition

Reply:
Also, taking into consideration that game could be in the future extended to more players it is worth to introduce _winner field, instead of DidLost property